src/rules/prefer-to-have-value.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @fileoverview prefer toHaveAttribute over checking  getAttribute/hasAttribute
+ * @author Ben Monro
+ */
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+export const meta = {
+  docs: {
+    category: "jest-dom",
+    description: "prefer toHaveValue over checking  element.value",
+    url: "prefer-to-have-value",
+    recommended: false,
```

Comment:
Why not recommend this rule? Seems safe and it's still fixible.

Reply:
Wanted to vet it a bit with the new smoke test feature we added. But yeah this is the plan